Makefile

```
@@ -9,41 +9,33 @@ vap_test_files:=\
+
+kubescape-rego-download:
+	mkdir -p public
+	curl -L https://github.com/kubescape/regolibrary/releases/download/v2/frameworks \
+	    | jq '.[].controls = []' > public/frameworks.json
+	curl -L https://github.com/kubescape/regolibrary/releases/download/v2/controls \
+	    -o public/controls.json
+	curl -L https://github.com/kubescape/regolibrary/releases/download/v2/rules \
+	    -o public/rego-rules.json
 
-download: wasm-download kubescape-download controls-download
+download: wasm-download kubescape-cel-admission-library-download kubescape-rego-download
```

checkmake (0.3.2)

[warning] Target "download" should be declared PHONY.

(phonydeclared)